webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -388,13 +535,14 @@ def update_info_text(
         ):
             return []
         info_text = []
-        ensemble = load_ensemble(parent, ensembles[0])
         active_response = active_resp_param["response"]
-        x_axis = ensemble.responses[active_response].axis
-        formatted_active_index = _format_index_value(
-            x_axis, corr_xindex.get(active_response, 0)
-        )
-        active_index_text = str(formatted_active_index)
+
+        selected_obs_indexes = selected_obs.get(active_response, None)
+        active_index_text = "NA"
+        if selected_obs and selected_obs_indexes:
+            active_index = corr_xindex.get(active_response) or selected_obs_indexes[0]
```

Comment:
I think this piece of logic should be moved to the function that outputs `corr_xindex` - the default value, when no observation is chosen, for the active index, is the first observation.

the same logic is repeated in line 495 f.

Reply:
Very good suggestion!

Reply:
: )